ds-caselaw-ingester/tests.py

```diff
@@ -717,21 +738,6 @@ def test_unpublish_updated_judgment(self, api_client, v2_ingest):
     def test_user_agent(self):
         assert "ingester" in lambda_function.api_client.session.headers["User-Agent"]
 
-    @patch("os.path.exists", return_value=True)
-    @patch("os.getenv", return_value="")
-    def test_unquote_s3(self, getenv, os):
```

Comment:
I think this test is still useful -- it's checking that we correctly url-decode S3 filenames.

We can inline the `from_event` or use AWS Powertools if you want.

Reply:
(this is the only thing I think is a blocker)